x-pack/test/api_integration/apis/uptime/rest/fixtures/browser_monitor.json

```diff
@@ -37,5 +37,6 @@
   "throttling.latency": "20",
   "throttling.config": "5d/3u/20l",
   "locations": [],
-  "name": "Test HTTP Monitor 03"
+  "name": "Test HTTP Monitor 03",
+  "namespace": "test namespace"
```

Comment:
Namespace can't contain space, so it's weird to see it like this in test fixtures.

Reply:
Agree, missed to review this one after we discussed content validation, will fix